### DISCUSSION TITLE
Docker logs on admin page

### DISCUSSION THREAD

**Discussion by superintendent2521** [diced/zipline#301] [Ideas]
either on its own page, or under account settings on administrator, administrator accounts should be able to see recent docker logs and upload them to the text service that zipline has built in

**Comment by diced** [comment]
The ability to view docker logs would be quite trivial, and would be irrelevant to Zipline's use case. It would just be easier to do `docker-compose logs`, or `docker logs`.

**Reply by superintendent2521** [reply]
gonna open this one back up due to the fact some people might not be able to SSH and view logs, and with the ability of having that be on the admin panel, having the API functionality for making, lets say a discord bot that pipes all logs to a channel would be quite nice
